src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -166,6 +175,7 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
 
   // TODO: Remove this function and use geth for lastL1BlockNumber!
   private async _updateLastL1BlockNumber() {
+    this.log.warn('Calling _updateLastL1BlockNumber...')
```

Comment:
Why is this a warning?

Reply:
^oo this is another spot i wanted an opinion on, i thought of putting warnings for all the functions we want to deprecate cc @karlfloersch @tynes whether we'll do that for this soon?

Reply:
@annieke shoot this function should be removed entirely — it was only used when Geth did not return an L1BlockNumber field. At this point this is dead code